lib/config/configuration.go

```diff
@@ -1028,6 +1028,12 @@ func applyProxyConfig(fc *FileConfig, cfg *service.Config) error {
 		cfg.Proxy.PeerPublicAddr = *addr
 	}
 
+	if fc.Proxy.IDP.SAMLIdP.Enabled() {
+		cfg.Proxy.IDP.SAMLIdP.Enabled = true
+	}
+
+	cfg.Proxy.IDP.SAMLIdP.BaseURL = fc.Proxy.IDP.SAMLIdP.BaseURL
```

Comment:
It's odd that this line says IDP several times with inconsistent casing. Can we decide on IDP or IdP and use it consistently?

Reply:
It's a linting issue -- I'd prefer to stick with IdP just due to its technical correctness, but how do you feel about adding in `nolint` directives everywhere this crops up?

Basically anywhere I've written `IDP` is because the linter complained about it.

Reply:
I've added in `nolint` everywhere to handle this. I tried configuring the `revive` `var-naming` plugin in golangci-lint, but even with the linting relaxed, it doesn't like the mixed case initialism. Let me know what you think.